grobid-core/src/main/java/org/grobid/core/engines/tagging/GenericTaggerUtils.java

```diff
@@ -73,8 +79,8 @@ public static String getPlainLabel(String label) {
     }
 
     public static boolean isBeginningOfEntity(String label) {
-        return StringUtils.startsWith(label, START_ENTITY_LABEL_PREFIX)
-            || StringUtil.startsWith(label, START_ENTITY_LABEL_PREFIX_ALTERNATIVE)
-            || StringUtil.startsWith(label, START_ENTITY_LABEL_PREFIX_ALTERNATIVE_2);
+        return StringUtils.startsWith(label, TaggingLabels.GROBID_START_ENTITY_LABEL_PREFIX)
+            || StringUtil.startsWith(label, TaggingLabels.IOB_START_ENTITY_LABEL_PREFIX)
+            || StringUtil.startsWith(label, TaggingLabels.ENAMEX_START_ENTITY_LABEL_PREFIX);
```

Comment:
This method will fail with `IOB_INSIDE_LABEL_PREFIX` returning `true` for `isBeginningOfEntity()` while it is not the beginning of an entity. Problem  `IOB_INSIDE_LABEL_PREFIX` == `GROBID_START_ENTITY_LABEL_PREFIX` == "I-".
Normally this method should not be called with IOB labels... but not 100% sure.

Reply:
Yes, it has been used for grobid-ner and uses only on Grobid labels, for what I can remember. However, we might want to 
 - split the two into Grobid and IOB 
 - create a new method just for IOB (<- My preference, since by default we are talking about grobid stuff?)

Reply:
Something like: 
```
    public static String getPlainIOBLabel(String label) {
        return isBeginningOfEntity(label) ? StringUtil.substring(label, 2) : label;
    }

    public static boolean isBeginningOfIOBEntity(String label) {
        return StringUtil.startsWith(label, TaggingLabels.IOB_START_ENTITY_LABEL_PREFIX)
            || StringUtil.startsWith(label, TaggingLabels.ENAMEX_START_ENTITY_LABEL_PREFIX);
    }

    // I-<citation> --> <citation>
    // <citation> --> <citation>
    public static String getPlainLabel(String label) {
        return isBeginningOfEntity(label) ? StringUtil.substring(label, 2) : label;
    }

    public static boolean isBeginningOfEntity(String label) {
        return StringUtils.startsWith(label, TaggingLabels.GROBID_START_ENTITY_LABEL_PREFIX)
            || StringUtil.startsWith(label, TaggingLabels.ENAMEX_START_ENTITY_LABEL_PREFIX);
    }
```

Reply:
In `grobid-ner` there is a duplication of the same method actually